.gitignore

```diff
@@ -15,6 +15,7 @@ tools
 examples
 sqltoolsservice
 coverage
+coverage-e2e
```

Comment:
Maybe just put the coverage inside the coverage folder to keep the root dirs cleaner.